README.md

```diff
@@ -268,7 +269,7 @@ jobs:
 ### Configuring gcloud
 
 This example demonstrates using this GitHub Action to configure authentication
-for the `gcloud` CLI tool. Note this does **NOT** work for the `gsutil` tool.
+for the `gcloud` CLI tool.
```

Comment:
Should we showcase this with setup-gcloud instead?

Reply:
Using a service account key JSON with `auth` will still work with gsutil and bq, just not WIF

Reply:
Wrong line 😅  I meant in https://github.com/google-github-actions/auth/blob/419a2c3bfece44e8b9089e3e3012b1b13ac53ec7/README.md?plain=1#L299-L307 if we could showcase using setup-gcloud after `auth` step rather than running `gcloud auth login --brief --cred-file`

Reply:
Sorry, I'm still not sure I'm following. Wanna open a PR against `sethvargo/readme`?

Reply:
Opened https://github.com/google-github-actions/auth/pull/101

Reply:
Merged